### DISCUSSION TITLE
Issue with iPXE UEFI & sanboot

### DISCUSSION THREAD

**Discussion by lucziv** [ipxe/ipxe#828] [General]
Could you help me to understand why I cannot boot from ISO images?

(iPXE 1.21 Configuration)

_:Linux
echo Linux OS
sanboot http://192.168.0.1/linux.iso || goto failed
goto start_

What I am missing?

I have attached Ubuntu 22 and Fedora 37 boot errors

![Ubuntu](https://user-images.githubusercontent.com/119680612/205310284-d046ab06-3169-4000-b666-61dae5f69023.png)

![Fedora](https://user-images.githubusercontent.com/119680612/205310620-16f303e1-0ad9-44e2-9378-e0d0977d309c.jpg)

**Comment by NiKiZe** [comment]
Does the iso have iSCSI and iBFT support?
Does it say anyware that these Isos are iSCSI compatible.

This was written based on the assumption that iSCSI was actually used.
sanboot with file over http will ONLY work if it loads all files it needs before booting any kernel. (Or you are staying in DOS or EFI land)

**Reply by lucziv** [reply]
Good question. I have downloaded these ISO's from official web sites.

What I should to do to make all of them compatible?

**Reply by NiKiZe** [reply]
Arch has some documentation for iBFT boot https://wiki.archlinux.org/title/ISCSI/Boot

The iscsi_bft modules and friends needs to be loaded and used at boot.
Normally you would not boot ISOs, instead use their instructions for netboot.

**Reply by lucziv** [reply]
Thanks!
